### PR TITLE
Add Laravel 8 Support

### PR DESCRIPTION
**Pull request by shahvirag**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,12 +22,12 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
+        "php": ">=7.3",
```
Comment (atymic) [inline composer.json]:
```suggestion
        "php": ">=7.2",
```
Can leave this at 7.2

**Review by atymic — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (shahvirag) [reply composer.json]: Yeah Sure. for version below below 8.x.

**Comment by shahvirag** [comment]
Need help on github action setup.
> Hot to skip laravel 8.0 on with PHP 7.2?

**Comment by atymic** [comment]
>Hot to skip laravel 8.0 on with PHP 7.2?

Just remove it 👍

**Review by atymic — APPROVED** [review]

**Comment by atymic** [comment]
Seems there is a syntax error, could you check?
image

**Pull request merged**